spikeinterface/sorters/ironclust/ironclust.py

```diff
@@ -34,7 +34,7 @@ class IronClustSorter(BaseSorter):
     ironclust_path: Union[str, None] = os.getenv('IRONCLUST_PATH', None)
 
     requires_locations = True
-    docker_requires_gpu = True
+    docker_requires_gpu = False
```

Comment:
@bendichter can you reset this to True? I refactored the whole thing here: https://github.com/SpikeInterface/spikeinterface/pull/674

Reply:
```suggestion
    docker_requires_gpu = True
```